lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1175,71 +1243,55 @@ def compute_severity(resource_type, record, entry_common_labels)
       end
     end
 
-    NANOS_IN_A_SECOND = 1000 * 1000 * 1000
+    def set_log_entry_field(record, entry, field_name, cfg)
+      payload_key = instance_variable_get(cfg[:payload_key])
+      return unless record[payload_key].is_a?(Hash)
+      subfields = {}
+      cfg[:fields].each do |key, destination_key, cast_fn|
+        value = record[payload_key][key]
+        next if value.nil?
+        unless cast_fn.nil?
+          begin
+            value = send(cast_fn, value)
+          rescue TypeError
```

Comment:
actually, no, because the rescue at the end of the method completely stops trying to set anything on the entry, so an invalid latency would drop the entire `httpRequest` part of the payload (and even worse, the `httpRequest` would not even be in the jsonPayload anymore because we already deleted the other fields